lib/prefix.h

```diff
@@ -284,6 +289,9 @@ extern void masklen2ip6 (const int, struct in6_addr *);
 extern void str2in6_addr (const char *, struct in6_addr *);
 extern const char *inet6_ntoa (struct in6_addr);
 
+extern int str2mac(const char *str, char *mac);
+extern char *mac2str(char *mac, char *buf, int size);
```

Comment:
For consistency with the rest of the prefix utils, shouldn't this take prefix or prefix_eth as arguments?